zulip_bots/zulip_bots/bots/ping/doc.md

```diff
@@ -0,0 +1,29 @@
+# Ping Bot
+
+This bot allows users to check availability of a given website or IP
+address across 10 random locations using the free API provided by
+[check-host.net](https://check-host.net). The API documentation can be
+found [here](https://check-host.net/about/api).
+
+### Example Usages
```

Comment:
Can you name this section "Usage"? (We try to be a little more consistent about to bot docs).